src/blocks/IndexHeader.vue

```diff
@@ -45,15 +45,15 @@
                 :class="{ opened: dropdownOpened }"
               >
                 <a class="dropDownHandler linkItem _position-top-0" @click.prevent="handleDropdown">
-                  zkTools
+                  RIF Aggregation Tools
```

Comment:
Maybe we should remove the checkout dapp until it's not released

Reply:
Ok, just comment the line until we launch it